lib/menu.rb

```diff
@@ -0,0 +1,13 @@
+class Menu
+ # open/close principle - open to extension closed to modification
+  attr_reader :dishlist
+  def initialize(dishlist)
```

Comment:
This is great - injecting the list of dishes dynamically instead of hard-coding them.